packages/core/src/components/ChoiceList/Select.jsx

```diff
@@ -82,6 +84,10 @@ Select.propTypes = {
   name: PropTypes.string.isRequired,
   onBlur: PropTypes.func,
   onChange: PropTypes.func,
+  /**
+   * Set the max-width of the input either to `'small'` (maps to `ds-c-field--small` class) or `'medium'` (maps to `ds-c-field--medium` class)
```

Comment:
I dunno if we need to specify what class these match up to, but I don't feel strongly either way.